### ISSUE TITLE
Execution failed for task ':transformClassesWithPreJackPackagedLibrariesForDebug'

### ISSUE THREAD

**Issue by mushishi78** [EddyVerbruggen/cordova-plugin-safariviewcontroller#119]
The following error happens when trying to build:

```
Execution failed for task ':transformClassesWithPreJackPackagedLibrariesForDebug'.
> java.lang.NullPointerException (no error message)
```

I've made a minimal reproduction: [mushishi78/jack-null](https://github.com/mushishi78/jack-null)

The output is here: [output.txt](https://raw.githubusercontent.com/mushishi78/jack-null/master/output.txt)

I posted about on a previous thread, but decided that it deserved a new issue: https://github.com/EddyVerbruggen/cordova-plugin-safariviewcontroller/issues/114

On that thread, @ish2028 had the same issue https://github.com/EddyVerbruggen/cordova-plugin-safariviewcontroller/issues/114#issuecomment-411469042

Any and all help welcome.

**Comment by mushishi78** [comment]
Seems to have been fixed by updating the android version:

```
cordova platform remove android
cordova platform add android@7.X.X
```

**Comment by biswas123** [comment]
Is there any way to solve this issue without upgrading cordova android  version? I want this fixed on 6.3.0
